### DISCUSSION TITLE
Add a video or screenshots about this compositor

### DISCUSSION THREAD

**Discussion by 0xDEAD-C0DE** [StrataWM/strata#73] [General]
Good afternoon. I was browsing the Smithay repository when I found this Wayland Compositor. I have tried Hyprland and I am currently trying Niri because is built with Rust. I wanted to try Strata but did not because the lack of visual information. A short video or screenshots will give Strata more attention.

Thank you for your hard work little cerebrito :) It is a expression we use in my country. Not implying that your brain is small, but literally stating that you are a young individual and very intelligent.

**Comment by anantnrg** [comment]
> Thank you for your hard work little cerebrito :) It is a expression we use in my country. Not implying that your brain is small, but literally stating that you are a young individual and very intelligent.

Thanks for the compliment! The reason i haven't added screenshots is because Strata is still very incomplete and lacks a lot of basic features. My current plan is to implement the Udev backend so it can actually be launched from a TTY without an existing WM, then cleanup the Lua API. After this I'll add some screenshots or a video. But it might take a few weeks. The two of us working on this actively are busy with school (exams) + i have a lot of pending projects. But it will be done soon. 

Cheers :)
